src/dashboard/src/pages/CommonComponents/DLTSSnackbar.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {Fragment} from "react";
+import {
+  createStyles,
+  makeStyles,
+  Snackbar,
+  SnackbarContent,
+  Theme
+} from "@material-ui/core";
+import {blue, green, red} from "@material-ui/core/colors";
+
+
+interface SnackbarProps {
+  children?: React.ReactNode;
+  message: string;
+  openKillWarn?: boolean | false;
+  openPauseWarn?: boolean | false;
+  openResumeWarn?: boolean | false;
+  openUpatePriorityWarn?: boolean | false;
+  openApproveWarn?: boolean | false;
+  autoHideDuration?: number | 1000;
+  handleWarnClose?: any;
+}
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    success: {
+      backgroundColor: green[600],
+    },
+  })
+);
+export const DLTSSnackbar = (props: SnackbarProps) => {
+  const classes = useStyles();
+  const { children,handleWarnClose,autoHideDuration,message,openKillWarn,openResumeWarn,openApproveWarn,openPauseWarn,openUpatePriorityWarn } = props;
```

Comment:
Seems it only used in job status notification? Rename it as "Job status snackbar" might be better.